content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+| A5    | Serie 6      | Class CL  | Astra         |
++-------+--------------+-----------+---------------+
+| A6    | Serie 7      | Class CLS | AstraGTC      |
++-------+--------------+-----------+---------------+
+| A7    | Serie Hybrid | Class E   | Combo Tour    |
++-------+--------------+-----------+---------------+
+| A8    | Serie M      | Class GL  | Corsa         |
++-------+--------------+-----------+---------------+
+| Q3    | Serie X      | Class GLK | Insignia      |
++-------+--------------+-----------+---------------+
+| Q5    | Serie Z4     | Class M   | Meriva        |
++-------+--------------+-----------+---------------+
+| TT    |              | Class R   | Mokka         |
++-------+--------------+-----------+---------------+
+|       |              | Class S   | Zafira        |
++-------+--------------+-----------+---------------+
+|       |              | Class SLK | Zafira Tourer |
++-------+--------------+-----------+---------------+
+|       |              | Class SLS |               |
++-------+--------------+-----------+---------------+
+
+.. _configuration/add-model:
+
+Add a new model
+---------------
+
+New vehicle models can easily be added to the database. To add a new model, navigate to
+:menuselection:`Fleet app --> Configuration --> Vehicle Models`. Click :guilabel:`Create` and a
+vehicle model form will load. Enter the following information on the form, then click
+:guilabel:`Save`. Be advised, some fields are specific to Belgian based companies, so not all fields
+or sections may be visible depending on the location of the company.
+
+- :guilabel:`Model name`: Enter the model name in the field.
+- :guilabel:`Manufacturer`: Select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  & Edit`.
+- :guilabel:`Vehicle Type`: Select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application. Adding additional vehicle types is not possible as it
+  will affect payroll.
+- :guilabel:`Category`: Select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create
+  (new category)`.
+
+.. note::
+   When the manufacturer is selected, the logo for the manufacturer will automatically load in the
+   image box in the top right corner.
+
+Information tab
+~~~~~~~~~~~~~~~
+
+Model
+*****
+
+- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
+- :guilabel:`Color`: Enter the color of the vehicle.
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
```

Comment:
This is covered in the other fleet doc? is this the same info?

Reply:
This is in the new vehicle doc, yes. I noted on there that I'll let @StraubCreative decide if I should remove that section form that doc, or remove it form here and link to that doc, or keep them in both,